app/src/main/java/org/schabi/newpipe/util/urlfinder/PatternsCompat.java

```diff
@@ -18,159 +18,13 @@
 
 package org.schabi.newpipe.util.urlfinder;
```

Comment:
I don't think you can just remove things from this file at random. This file was taken from AOSP and modified, as the licence explains (though a comment would be better), so any change should be documented I guess.